ipc/tests/blocking_client.rs

```diff
@@ -70,7 +71,4 @@ fn test_blocking_client() {
 
     let mut f = f.into_instance().unwrap();
     writeln!(f, "test").unwrap(); // file should still be writeable
-
-    drop(transport);
-    worker.join().unwrap();
```

Comment:
I'm curious about this change, will this create flakiness? E.g. if this thread stays alive and is concurrent with other tests.

Reply:
You're right - I changed the test to drop the tokio runtime on exit. :bow:

Reply:
Ah, I think I see -- it happens implicitly when `rt` is dropped at the end of the method, is that it?